app/code/Magento/Directory/Setup/Patch/Data/InitializeDirectoryData.php

```diff
@@ -491,15 +491,15 @@ public function apply()
             ['ES', 'Vizcaya', 'Vizcaya'],
             ['ES', 'Zamora', 'Zamora'],
             ['ES', 'Zaragoza', 'Zaragoza'],
-            ['FR', 1, 'Ain'],
-            ['FR', 2, 'Aisne'],
-            ['FR', 3, 'Allier'],
-            ['FR', 4, 'Alpes-de-Haute-Provence'],
-            ['FR', 5, 'Hautes-Alpes'],
-            ['FR', 6, 'Alpes-Maritimes'],
-            ['FR', 7, 'Ardèche'],
-            ['FR', 8, 'Ardennes'],
-            ['FR', 9, 'Ariège'],
+            ['FR', 01, 'Ain'],
```

Comment:
01 is integer in octal notation (see in https://www.php.net/manual/en/language.types.integer.php) 
probably you want to treat is as string
```suggestion
            ['FR', '01', 'Ain'],
```

Reply:
Thanks